<launch>
<!--  <arg name="robot_name" />-->
<!--  <param name="robot_name" type="str" value="$(arg robot_name)" />-->
<!--  <param name="xbee_port" value="$(arg xbee_port)"/>-->
  <node name="xbee_interface_node" pkg="xbee_communication" type="xbee_interface_node.py" />
</launch>

